include/picolibrary/adafruit/pid781.h

```diff
@@ -97,6 +101,242 @@ struct Cursor_Position {
+            return result.error();
+        } // if
+
+        return std::size_t{ 1 };
+    }
+
+    /**
+     * \brief Write the formatted picolibrary::Adafruit::PID781::Custom_Character to the
+     *        stream.
+     *
+     * \param[in] stream The stream to write the formatted
+     *            picolibrary::Adafruit::PID781::Custom_Character to.
+     * \param[in] custom_character The picolibrary::Adafruit::PID781::Custom_Character to
+     *            format.
+     *
+     * \return The number of characters written to the stream.
+     */
+    auto print( Reliable_Output_Stream & stream, Adafruit::PID781::Custom_Character const & custom_character ) const noexcept
+        -> std::size_t
+    {
+        stream.put( to_underlying( custom_character.id ) );
+
+        return std::size_t{ 1 };
+    }
+};
+
+/**
+ * \brief picolibrary::Adafruit::PID781::Custom_Character::ID output formatter.
+ */
+template<>
+class Output_Formatter<Adafruit::PID781::Custom_Character::ID> {
+  public:
+    /**
+     * \brief Constructor.
+     */
+    constexpr Output_Formatter() noexcept = default;
+
+    /**
+     * \brief Constructor.
+     *
+     * \param[in] source The source of the move.
+     */
+    constexpr Output_Formatter( Output_Formatter && source ) noexcept = default;
+
+    /**
+     * \brief Constructor.
+     *
+     * \param[in] original The original to copy.
+     */
+    constexpr Output_Formatter( Output_Formatter const & original ) noexcept = default;
+
+    /**
+     * \brief Destructor.
+     */
+    ~Output_Formatter() noexcept = default;
+
+    /**
+     * \brief Assignment operator.
+     *
+     * \param[in] expression The expression to be assigned.
+     *
+     * \return The assigned to object.
+     */
+    constexpr auto operator=( Output_Formatter && expression ) noexcept -> Output_Formatter & = default;
+
+    /**
+     * \brief Assignment operator.
+     *
+     * \param[in] expression The expression to be assigned.
+     *
+     * \return The assigned to object.
+     */
+    constexpr auto operator   =( Output_Formatter const & expression ) noexcept
+        -> Output_Formatter & = default;
+
+    /**
+     * \brief Write the formatted picolibrary::Adafruit::PID781::Custom_Character::ID to
+     *        the stream.
+     *
+     * \param[in] stream The stream to write the formatted
+     *            picolibrary::Adafruit::PID781::Custom_Character::ID to.
+     * \param[in] id The picolibrary::Adafruit::PID781::Custom_Character::ID to format.
+     *
+     * \return The number of characters written to the stream if the write succeeded.
+     * \return An error code if the write failed.
+     */
+    auto print( Output_Stream & stream, Adafruit::PID781::Custom_Character::ID id ) const noexcept
+        -> Result<std::size_t>
+    {
+        auto result = stream.put( to_underlying( id ) );
+        if ( result.is_error() ) {
+            return result.error();
+        } // if
+
+        return std::size_t{ 1 };
+    }
```

Comment:
This overload should be removed since the driver will only support reliable output streams.